josh-mongo/__tests__/index.test.js

```diff
@@ -0,0 +1,203 @@
+
+const Provider = require('../index.js');
+
+const provider = new Provider({ name: 'InMemoryJosh' });
+
+test('Database instance is valid', () => {
+  expect(provider).not.toBe(null);
+  expect(provider.name).toBe('inmemoryjosh');
+});
+
+test('Database can be initialized', async () => {
+  await provider.init();
+});
+
+test("Database can be closed", () => {
```

Comment:
```suggestion
test('Database can be closed', () => {
```
keep the single quote consistency here

Reply:
Thanks!